fluent-package/debian/control

```diff
@@ -1,4 +1,4 @@
-Source: td-agent
+Source: fluent-package
```

Comment:
In this PR I don't want to rename the package name yet.
I want rename only the sub directory name of fluent-package-builder's source.

Reply:
Ops, sorry.